packages/expo-doctor/src/utils/explainDependencies.ts

```diff
@@ -31,7 +32,7 @@ async function explainAsync(
     return JSON.parse(stdout);
   } catch (error: any) {
     if (isSpawnResult(error)) {
-      if (error.stderr.match(/npm ERR! No dependencies found matching/)) {
+      if (error.stderr.match(/No dependencies found matching/)) {
```

Comment:
Maybe nice to add a (e2e) test to support this, and warn us if this breaks in the future again 😄